### PR TITLE
docs(spanner): changes to support sphinx new release

### LINKED ISSUE #50: Investigate new Sphinx release changes
Sphinx has a new release: [3.0.0](https://www.sphinx-doc.org/en/master/changes.html#release-3-0-0-released-apr-06-2020)

This release has caused the docs generation to fail due to issues in CHANGELOG.md. The root cause in the CHANGELOG should be found and fixed so the library can continue to rely on the most recent update.

If this proves difficult, the version can be temporarily pinned to 2.2.4 in the interim.

### PR DESCRIPTION
**Pull request by HemangChothani**
Fixes #50 

Hope this Draft PR helps to fix issue quickly.

- This draft PR includes few gapic `.rst` files changes which i can't address so can't open Public PR, but changed that gapic `.rst` files for just references.

### CODE REVIEW

**Comment by HemangChothani** [comment]
@larkee Please let me know could i make changes for docs/gapic/*.rst files manually?

**Comment by larkee** [comment]
Good news! Those files aren't generated so this PR looks good to me :)

I understand the confusion. I had to run the synthtool to confirm. I suspect the reason these files are in folder named gapic is to match the folder structure of the files they're generating documentation for.

File: CHANGELOG.md [hunk 1]
```diff
@@ -97,12 +99,12 @@ Return sessions from pool in LIFO order. ([#9454](https://github.com/googleapis/
...
 
 ### Implementation Changes
-- Add backoff for `run_in_transaction' when backend does not provide 'RetryInfo' in response. ([#8461](https://github.com/googleapis/google-cloud-python/pull/8461))
+- Add backoff for `run_in_transaction` when backend does not provide 'RetryInfo' in response. ([#8461](https://github.com/googleapis/google-cloud-python/pull/8461))
 - Adjust gRPC timeouts (via synth). ([#8445](https://github.com/googleapis/google-cloud-python/pull/8445))
 - Allow kwargs to be passed to create_channel (via synth). ([#8403](https://github.com/googleapis/google-cloud-python/pull/8403))
 
 ### New Features
-- Add 'options_' argument to clients' 'get_iam_policy'; pin black version (via synth). ([#8659](https://github.com/googleapis/google-cloud-python/pull/8659))
+- Add 'options' argument to clients 'get_iam_policy'; pin black version (via synth). ([#8659](https://github.com/googleapis/google-cloud-python/pull/8659))
```
Comment (larkee) [inline CHANGELOG.md:107]: Actually, is it possible at all to keep the underscore? Ideally we want to match the parameter name. If not, this is fine.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,5 @@
+
```
Comment (larkee) [inline CHANGELOG.md:1]: Remove these new lines.

**Pull request merged**